node_modules/node-red-node-email/61-email.html

```diff
@@ -42,6 +42,10 @@
         <input type="text" id="node-input-subject" placeholder="Message title">
     </div>
     <div class="form-row">
+        <label for="node-input-body"><i class="fa fa-envelope"></i> <span data-i18n="email.label.body"></span></label>
+        <input type="text" id="node-input-body" placeholder="Message body">
```

Comment:
If this is actually a reference to a variable being set on the flow, I think we should use the typed field def from node-red, having only the 'msg.' type being allowed (would make for better normalization of the fields, which helps the user to comprehend the nature of the information being inputted)